apps/codeforafrica/src/lib/index.js

```diff
@@ -776,18 +834,22 @@ export function getOpportunities(options) {
   return paginateResults(found, page, pageSize);
 }
 
-function getOpportunitiesTags(options = { includeAll: true }) {
-  const tags = Array.from(
-    new Set(opportunities?.flatMap((o) => o.tags || []))
-  ).sort();
+async function getOpportunitiesPageStaticProps(options) {
+  const allOpportunities = await getPostsByPrimaryTag("opportunities", options);
 
-  if (options?.includeAll) {
-    return [ALL_TAG, ...tags];
-  }
-  return tags;
-}
+  // append /opportunities/ to the slug && transform published date to ISO format
+  allOpportunities.map((a) => {
+    const { slug: originalSlug, ...other } = a;
+    const slug = `/opportunities/${originalSlug}`;
+    return { slug, ...other };
+  });
+
+  const allTags = allOpportunities
+    .map((article) => article.tags)
+    .flat(Infinity);
```

Comment:
What is `.flat(Infinity)`? Doesn't `flatMap` work?
```suggestion
  const allTags = allOpportunities
    .flatMap((article) => article.tags);
```